lib/exportJob.js

```diff
@@ -350,9 +378,9 @@ class ExportJob extends EventEmitter {
       }
     } else {
       pageDim = pageDimensions[pageSize]
+      // Flip if landscape
+      pageDim = landscape ? {x: pageDim.y, y: pageDim.x} : pageDim
```

Comment:
I don't totally follow what's going on here, but would this be helpful to include as a comment inside the code?